iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/executable/ExecutableManager.java

```diff
@@ -227,6 +228,8 @@ protected void saveToDir(ByteBuffer byteBuffer, String destination) throws IOExc
     try {
       Path path = Paths.get(destination);
       Files.deleteIfExists(path);
+      // Ensure the target directory exists.
+      SystemFileFactory.INSTANCE.makeDirIfNecessary(destination);
```

Comment:
> Ok ... so finally I think I've ironed out all quirks ... however no Idea why the integration tests only seldomly seem to pass ... haven't quite detected a pattern yet.

The failed integration tests are related to Trigger. And the reason is what I mentioned in your previous PR #10709. The Line 232 is unnecessary.

Reply:
Thanks for re-reporting that ... now that you mention it, you did report that on the old PR, that I dropped, because I wanted to keep the noise on the list to a mimimum, but in the end couldn't completely as my fork didn't run the Workflows :-( ... I wasn't ignoring you intentionally ;-)